examples/plot_grid_search_census.py

```diff
@@ -229,3 +229,20 @@ def classifier(X): return m.predict(X)
 #
 # In a real example, we would pick the model which represented the best trade-off
 # between accuracy and disparity given the relevant business constraints.
+#
+# %% [markdown]
+# Comparing models easily
+# -----------------------
+# Fairlearn also provides functionality to compare models much more easily. 
+
+# %%
+
+# Plot model comparison
+plot_model_comparison(
```

Comment:
>     * I think the example I made is a bit boring, any tips?

Hmm, for one I think it's nice that you're building on the existing case rather than copy-pasting without real changes. I don't think it's boring as such, but eventually we're hopefully not using the adult census dataset anymore. #907  will introduce a synthetic dataset for these purposes. As we integrate this model comparison into real world use case examples it'll get much more interesting, of course.